.gitignore

```diff
@@ -16,3 +16,5 @@ packages/
 OALLexer.tokens
 .factorypath  
 .vscode 
+.checkstyle
```

Comment:
This checkstyle generated by eclipse?

Reply:
> This checkstyle generated by eclipse?

Yes, generated by eclipse checkstyle plugin